fideslog/api/models/analytics_event.py

```diff
@@ -61,8 +61,8 @@ class AnalyticsEvent(BaseModel):
         None,
         description="For events submitted as a result of running CLI commands, the flags in use when the command was submitted. Omits flag values when they exist.",
     )
-    local_host: bool = Field(
-        False,
+    local_host: Optional[bool] = Field(
+        None,
         description="For events submitted as a result of making API server requests, `true` if the API server is running on the user's local host, otherwise `false`.",
```

Comment:
Do we need a None description. None if N/A?

Reply:
IMO, the beginning of the description ("For events submitted as a result of making API server requests...") strongly implies that this is only allowed to `None` for events that are _not_ API server requests. This is aligned with the descriptions of the other fields. If you're saying these are unclear, maybe we should separately update all of the field descriptions?

Reply:
ok that makes sense, I'm fine as is